### DISCUSSION TITLE
Generate android & iOS projects in a separated git repository

### DISCUSSION THREAD

**Discussion by alexsedlex** [ionic-team/capacitor#3393] [Q&A]
Hi everyone, we've been using Capacitor for a few months, and so far it fits every needs we have and it's awesome (see what we've done with it here http://amorel.page.nuiton.org/free-as-a-bird/web/mobile/capacitor-webapp-mobile-vue-js/).

I'm setting up Capacitor on a new project, and the project owner would really like to keep the android & iOS folders separated from the web source code, in a dedicated git repository.

We have a lot of work-aroundy ideas, but before setting this up is there a configuration feature in capacitor that I missed that would allow to configure the location of android & iOS folders (instead of using ./) ?

**Comment by imhoffd** [comment]
Surprisingly no, there isn't. I found this feature request for it: https://github.com/ionic-team/capacitor/issues/2294
